docs/_documentations/project-actions.md

```diff
@@ -6,35 +6,40 @@ keywords: run, open, import, show, restart, edit, build, logs, tools, project ac
 duration: 1 minute
 permalink: project-actions
 type: document
-order: 30
-parent: 
 ---
 
 # Project actions
 
-The following actions are available by right clicking on the project in the **Codewind Explorer** view. Most actions are only available if the project is enabled.
+The following actions are available by right-clicking the project in the **Codewind Explorer** view. Most actions function only if the project is enabled.
 
-Some actions open the default browser. If you find that the default browser cannot handle the content, change the default browser by navigating to **Window** > **Web Browser** and selecting a different browser from the list.
-
-- **Open Application**: Opens the application in the default browser. This action is only available when the application is running or debugging.
-- **Project Overview**: Opens the overview page for this project. You can use this to see information about the project or edit project settings.
-- **Metrics Dashboard**: Opens the application monitor in the default browser. Use this to monitor the activity and health of your application. This action is only available when the application is running or debugging.
-- **Performance Dashboard**: Opens the performance dashboard in the default browser. This action is available only when the application is running or debugging.
-- **Tekton Dashboard**: This option is available to you if you are using remote connections, and only if you installed pipelines. Use this option to open the dashboard.
-- **Open Container Shell**: Opens a shell into your application container. This action is only available when the container is active.
+Some actions open the default browser. If the default browser cannot handle the content, change the default browser.
 
+- **Open Application**: Opens the application in the default browser. This action functions only when the application is running or debugging. If you try to access it before running the application, a warning message appears.
+- **Project Overview**: Opens the overview page for the project. Use this action to see information about the project or edit project settings.
+- **Metrics Dashboard**: In Java, Node.js, and Swift applications, this action opens the Metrics Dashboard in the default browser. Use this action to monitor the activity and health of your application. This action is available only when the application is running or debugging.
```

Comment:
Metrics Dashboard - It repeats "this action" three consecutive times. I would suggest replacing at least one with a pronoun or an equivalent subject.

Reply:
For clarity, I would prefer to leave the repetition. For example, if we replaced "it" with one of the instances of "this action," it might be a little more unclear what "it" refers to, but there's no guesswork if we state what it is. :)